src/components/daily-dropoffs-line-chart.tsx

```diff
@@ -0,0 +1,112 @@
+import { max } from "d3-array";
+import { axisLeft, axisBottom } from "d3-axis";
+import { scalePoint, scaleLinear } from "d3-scale";
+import { VNode } from "preact";
+import { useRef, useState } from "preact/hooks";
+import { line as d3Line } from "d3-shape";
+import { formatWithCommas, formatAsPercent } from "../formats";
+import {
+  DailyDropoffsRow,
+  FunnelMode,
+  funnelSteps,
+  StepCount,
+  stepToTitle,
+  toStepCounts,
+} from "../models/daily-dropoffs-report-data";
```

Comment:
originally I had `models/daily-dropoffs-report` and `reports/daily-dropoffs-report` as the two files, but I remembered I hate doing things that way because it makes it hard to open the right file in my editor